test/parallel/test-vm-module-errors.js

```diff
@@ -250,12 +250,27 @@ async function checkExecution() {
   })();
 }
 
+// Check for error thrown when breakOnSigint is not a boolean for evaluate()
+async function checkInvalidOptionForEvaluate() {
+  await assert.rejects(async () => {
+    const m = new SourceTextModule('export const a = 1; export var b = 2');
+    await m.evaluate({ breakOnSigint: 'a-string' });
+  }, {
+    name: 'TypeError',
+    message:
+      'The "options.breakOnSigint" property must be of type boolean. \
```

Comment:
We usually avoid using `\` to continue strings on new lines. Not objecting, but if someone else does, you can switch to:

```js
'some string. ' +
'some other string.'
```

Reply:
@Trott ,
Thank you for the review. I'll update the PR today.